packages/bsky/src/views/index.ts

```diff
@@ -608,13 +609,14 @@ export class Views {
     }
     const post = this.post(item.post.uri, state)
     if (!post) return
-    return {
+    const view = {
```

Comment:
any reason for making this a var before returning? (guessing this is just cruft from rewriting something)